src/Aspire.Hosting/ExecutableResourceBuilderExtensions.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Dashboard;
```

Comment:
Lets fix this namespace. This shouldn't be dashboard related.

Reply:
I followed the prior art from `ConsoleLogsConfigurationExtensions` which is in this namespace.

Reply:
Is this API internal? If so I am fine and we can change it later.

Reply:
I changed it to Aspire.Hosting.ApplicationModel. Let me know if there is somewhere else you want it.